src/qibo/optimizers/gradient_based.py

```diff
@@ -0,0 +1,135 @@
+"""Gradient descent strategies to optimize quantum models."""
+
+from qibo import set_backend
+from qibo.backends import TensorflowBackend
+from qibo.config import log, raise_error
+from qibo.optimizers.abstract import Optimizer, check_options
+
+
+class TensorflowSGD(Optimizer):
+    def __init__(
+        self,
+        initial_parameters,
+        args=(),
+        loss=None,
```

Comment:
```suggestion
        loss=None,
        args=(),
```

Same order as the docstring